content/en/docs/setup/independent/control-plane-flags.md

```diff
@@ -22,8 +22,10 @@ The `extraArgs` field consist of `key: value` pairs. To override a flag for a co
 1.  Add the appropriate fields to your configuration.
 2.  Add the flags to override to the field.
 
+You can also add `extraVolumes` to mount additional volumes if necessary (e.g. including configuration files that are relevant to your cloud provider).
+
```

Comment:
please exclude this sentence for now.
we might refactor this document eventually.

Reply:
Okay it's done